users_service/models/error.go

```diff
@@ -0,0 +1,41 @@
+package models
+
+import (
+	"net/http"
+	"strings"
+
+	"users_service/resources"
+)
+
+type ErrorResponse struct {
+	Message string `json:"message"`
+}
+
+type ServiceError struct {
```

Comment:
I actually like how I did this and how it's used in the code. Let's consider making this (and probably serialization/deserialization helpers) common for all services.

Reply:
Not as useful as I thought, closing